### ISSUE TITLE
Gilbert's feedback

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/draft-guide-sessions#12]
- [x] is there a reason (is it a practice) to highlight the those words `sessions`, `session`, `cookies, `session persistence`, `session cache`?
- [x] using java11, failed to run test during `mvn install`
```
Running it.io.openliberty.guides.cart.CartSessionTest
Tests run: 3, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 0.639 sec <<< FAILURE! - in it.io.openliberty.guides.cart.CartSessionTest
testOneServer(it.io.openliberty.guides.cart.CartSessionTest)  Time elapsed: 0.63 sec  <<< ERROR!
java.lang.NoClassDefFoundError: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:753)
```
- [x] at the "Try what you’ll build" section, should give more detail instructions how to make a POST and GET requests?
- [ ] should use parent pom?
- [x] `<dependencies/>` 
  - should comment the testing dependencies
  - add BOM if it is required, 
- [x] maven-surefire-plugin and maven-failsafe-plugin should use 3.0.0-M1
- [x] ` <include>**/it/**</include>` should be changed to  `<include>**/it/**/*.java</include>` to exclude non java files in case
- [x] in the "Setting up multiple servers" section, better to have hotpot for "two local server instances"
- [ ] may need to explain what is server2 which is running the same app 
-  [x] in the "Building and running the application" section, after start the 2 servers, should provide instruction to make a POST request on server1 and make a GET request on server2, so that readers may see how thing really work (maybe the  "Try what you’ll build" section too)
- [x] Remind readers to stop servers either
   - in the end of  "Building and running the application" section
   - or before `mvn verify`
- [x] the "Contribute to this guide" section is a dup

**Comment by evelinec** [comment]
Alex had fixed the jdk11 failure via #13 
 - using java11, failed to run test during mvn install

**Comment by evelinec** [comment]
Regarding should use parent pom, actually, it's not applicable to this guide, as there are specific configs needed, ie, creating 2 servers, apps, and managing the starting and stopping of them. 

- should use parent pom?

**Comment by evelinec** [comment]
Regarding the point below, the paragraph seems clear that the app will create two "identical" servers that runs the same app. No change is made. 

-  may need to explain what is server2 which is running the same app

**Comment by evelinec** [comment]
About this point, the more detail results are added to the "Building and running the application" section, but not in the "Try what you'll build" section to keep it short. 

- in the "Building and running the application" section, after start the 2 servers, should provide instruction to make a POST request on server1 and make a GET request on server2, so that readers may see how thing really work (maybe the "Try what you’ll build" section too)

**Comment by evelinec** [comment]
Addressed the rest of the points in #17. @gkwan-ibm pls review. Thanks.
